static/web.js

```diff
@@ -553,6 +575,8 @@
         themes = document.getElementById("themes");
         editor = ace.edit("editor");
         set_result.editor = editor;
+        editor.$blockScrolling = Infinity;
```

Comment:
That var is also used [here](https://github.com/ajaxorg/ace/blob/759c65090f7ef50743d9ae4f5bfba3703fe2b663/lib/ace/autocomplete/popup.js#L215)

It looks like it's needed to be set in order to avoid showing a warning when calling setValue()  https://github.com/ajaxorg/ace/issues/2499